packages/block-library/src/list/test/edit.native.js

```diff
@@ -476,6 +477,10 @@ describe( 'List block', () => {
 			preventDefault() {},
 			keyCode: BACKSPACE,
 		} );
+		// Inner blocks batch store updates with microtasks.
+		// To avoid `act` warnings, we let queued microtasks to be executed.
+		// Reference: https://t.ly/b95nA
+		await act( async () => {} );
```

Comment:
I suppose this could eventually be replaced with `addBlock` if the helper retains its version of this.

Reply:
Yes, although this test case doesn't use `addBlock` but initial HTML ([reference](https://github.com/WordPress/gutenberg/pull/51303/files#diff-52bc11cb1e553aa6c760699ba925f3627336ce8d841939d4a9992726f60887dbR439-R457)).

Reply:
Ah, I misinterpreted the diff summary as being related to _adding_ an inner block. The reality is that this line relates to modifying an inner block already existing in the initial HTML. Apologies of the noise.